src/components/CreateTarget/index.js

```diff
@@ -59,9 +59,9 @@ const CreateTarget = ({
   );
 
   const onStartChatting = useCallback(
-    ({ matchedUser, matchConversation }) => {
+    (fullName, matchId) => {
       setShowMatch(false);
-      navigation.push(CHAT_SCREEN, { matchedUser, matchConversation });
+      navigation.push(CHAT_SCREEN, { userName: fullName, matchId });
```

Comment:
```suggestion
      navigation.navigate(CHAT_SCREEN, { userName: fullName, matchId });
```
Use navigate in case you already have an instance of this screen on the stack, so you not push again the screen